pyomo/core/tests/unit/test_set.py

```diff
@@ -4374,12 +4399,15 @@ def _validate(model, i, j):
         m.I2 = Set(validate=_validate)
         with LoggingIntercept(module='pyomo.core') as output:
             self.assertTrue(m.I2.add((0, 1)))
-            self.assertRegex(
-                output.getvalue().replace('\n', ' '),
-                r"DEPRECATED: OrderedScalarSet I2: 'validate=' callback "
-                r"signature matched \(block, \*value\).  Please update the "
-                r"callback to match the signature \(block, value\)",
-            )
+            # Note that we are not emitting a deprecation warning (yet)
+            # for scalar sets
+            # self.assertEqual(output.getvalue(), "")
+            #     output.getvalue().replace('\n', ' '),
+            #     r"DEPRECATED: OrderedScalarSet I2: 'validate=' callback "
+            #     r"signature matched \(block, \*value\).  Please update the "
+            #     r"callback to match the signature \(block, value\)",
+            # )
```

Comment:
Is this still supposed to be here?

Reply:
Sorry - this was leaving a breadcrumb (with no explanation).  Eventually we will (probably?) want to turn the deprecation warning back on, in which case this test will have to revert back.